dask/dataframe/tests/test_dataframe.py

```diff
@@ -1821,3 +1821,19 @@ def test_columns_assignment():
     ddf[['a', 'b']] = ddf2[['y', 'z']]
 
     eq(df, ddf)
+
+
+@pytest.mark.parametrize("idx", [
+    np.arange(100),
+    sorted(np.random.random(size=100)),
+    pd.date_range('20150101', periods=100)
+])
```

Comment:
Is it possible to add skipna as a parameter to the test

    @pytest.mark.parametrize('skipna', [True, False])

and then add `skipna=skipna` to all of the idxfoo calls?  That way we also test combinations like with axis=1 and with series